projects/storefrontlib/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.html

```diff
@@ -5,20 +5,23 @@
     [attr.aria-label]="
       'TabPanelContainer.tabPanelContainerRegion' | cxTranslate
     "
+    class="container"
   >
     <ng-container *ngFor="let component of components; let i = index">
       <ng-container *ngIf="component">
-        <button [class.active]="i === activeTabNum" (click)="select(i, $event)">
+        <button
+          [class.active]="i === activeTabNum"
+          (click)="select(i, $event)"
+          [attr.aria-expanded]="i === activeTabNum"
+        >
           {{
             component.title | cxTranslate: { param: tabTitleParams[i] | async }
           }}
+
+          <span class="accordion-icon" aria-hidden="”true”"></span>
```

Comment:
Wondering if this double quote is required: `aria-hidden="”true”"`
I think we should try with `aria-hidden="true"`

Reply:
thanks, looks like prettier did not catch it.